source/except.d.ts

```diff
@@ -1,3 +1,31 @@
+import {IsEqual} from './includes';
+
+/**
+Returns `never` if `Exclude` is strictly equal to `Key`.
```

Comment:
A description of the purpose of the type would be useful before the technical explanation.

Reply:
Done.